pkg/network/usm/monitor_test.go

```diff
@@ -953,26 +953,44 @@ func writeInput(c net.Conn, input []byte, timeout time.Duration) error {
 	}
 }
 
-var testHeaderFields = []hpack.HeaderField{
-	{Name: ":authority", Value: http2SrvAddr},
-	{Name: ":method", Value: "POST"},
-	{Name: ":path", Value: "/aaa"},
-	{Name: ":scheme", Value: "http"},
-	{Name: "content-type", Value: "application/json"},
-	{Name: "content-length", Value: "4"},
-	{Name: "accept-encoding", Value: "gzip"},
-	{Name: "user-agent", Value: "Go-http-client/2.0"},
+// generateTestHeaderFields generates a set of header fields that will be used for the tests.
+func generateTestHeaderFields(pathNeverIndexed, withoutIndexing bool) []hpack.HeaderField {
```

Comment:
You can add wrappers 

```suggestion
// generateTestHeaderFields generates a set of header fields that will be used for the tests.
func headersWithNeverIndexedPath() []hpack.HeaderField { return generateTestHeaderFields(...) }
func headersWithoutIndexingPath() []hpack.HeaderField { return generateTestHeaderFields(...) }
func defaultHeaders() []hpack.HeaderField { return generateTestHeaderFields(...) }
```

Reply:
that way you dont need to pass `false, false` for most of the instances or `true, false` or `false, true`